/**:
  ros__parameters:
    q_map: 0.3
    m_map: 0.2
    t_clip_min: 0.5
    t_clip_max: 3.0
    LU_table: "NUC1_pacejka"